lib/zhr_devs/event_store.ex

```diff
@@ -0,0 +1,3 @@
+defmodule ZhrDevs.EventStore do
```

Comment:
I think copy-paste explanation of what EventStore is from the documentation. I'd like for this codebase to be a self-contained intro I can point people to (including myself).